components/Container.js

```diff
@@ -1,7 +1,7 @@
 // Ensures the root container is always 100vw, min 100vh, and centers all children along the y-axis
 export default function Container({ children }) {
   return (
-    <div className="flex flex-col min-h-screen w-screen items-center">
+    <div className="flex flex-col min-h-screen w-screen max-w-full items-center">
```

Comment:
Here's where the action is. Just adding this `max-w-full` should be all it takes to fix the width.